ui/app/routes/vault/cluster/settings/auth/enable.ts

```diff
@@ -6,8 +6,13 @@
 import Route from '@ember/routing/route';
 import { service } from '@ember/service';
 
+import type { ModelFrom } from 'vault/vault/route';
+import type Store from '@ember-data/store';
```

Comment:
i noticed we've been importing the `Store` type 2 different ways. one is this way, and the other is like this:
https://github.com/hashicorp/vault/blob/077c70fc1fb90b4c4ffee4da2e1bed5f8496e671/ui/app/routes/vault/cluster/clients/counts.ts#L12

however importing the type from `@ember-data` works better since it recognizes that properties like `isDestroying` etc are indeed on that type. the alternative way of importing `Store` caused a bunch of type errors.

Reply:
That's strange, because in the `store.d.ts` file we're extending Store from `@ember-data/store` 🤔 Should be harmless here since we're not using any of the extended methods here, but smells like something wrong in our ts setup